pkg/gen/nodejs-templates/helm/v2/helm.ts

```diff
@@ -236,20 +236,13 @@ export class Chart extends yaml.CollectionComponentResource {
         resourcePrefix: string | undefined,
         dependsOn: pulumi.Resource[],
         defaultNamespace: string | undefined,
-        opts?: pulumi.ComponentResourceOptions,
     ): pulumi.Output<{ [key: string]: pulumi.CustomResource }> {
         // Rather than using the default provider for the following invoke call, determine the
         // provider from the parent if specified, or fallback to using the version specified
         // in package.json.
-        let invokeOpts: pulumi.InvokeOptions = {async: true};
-        if (opts?.parent) {
-            invokeOpts = {...invokeOpts, parent: opts.parent};
-        } else {
-            invokeOpts = {...invokeOpts, version: getVersion()};
-        }
+        let invokeOpts: pulumi.InvokeOptions = { async: true, version: getVersion() };
```

Comment:
Do we still also need to pass through a parent so that it uses the right provider instance?  Or does this invoke not actually care about the provider?  Even if it doesn’t now, should we pass the parent through regardless?

I actually don’t know for sure - so maybe input from @pgavlin would be good.

Reply:
I tried passing both args, and it was failing the same way (invoke returning an empty object `{}`). That may be a bug with the engine, but I don't know enough about it to be sure.

I think using the version makes sense.

Reply:
https://github.com/pulumi/pulumi/issues/3914

Reply:
> Even if it doesn’t now, should we pass the parent through regardless?

I think the version makes the most sense here as well. This invoke does not need the particular provider instance (yet, at least).